BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.1.0-preview2-02516-02
```

Comment:
This is to get MaxRetryCount defaulting back to 0, as we're not quite ready for auto-retry and that should be specified in upload-tests.proj

Reply:
Is this needed to fix the corefx build currently?